examples/demo-rollup/benches/rollup_config.toml

```diff
@@ -9,6 +9,8 @@ celestia_rpc_auth_token = "MY.SECRET.TOKEN"
 celestia_rpc_address = "http://localhost:11111/"
 # The largest response the rollup will accept from the Celestia node. Defaults to 100 MB
 max_celestia_response_body_size = 104_857_600
+# The maximum time to wait for a response to an RPC query against Celestia node. Defaults to 30 seconds.
+celestia_rpc_timeout_seconds = 30
```

Comment:
@dubbelosix, it looks to me like the config for the benches is identical to the one for the demo. Am I missing something? If not, should we think about deduplicating?

Reply:
Synced offline - we'll probably remove this shortly. Not going to block this PR on it tho!